Application/Sources/Content/PageViewModel.swift

```diff
@@ -342,6 +342,7 @@ extension PageViewModel {
         case showGrid
         case showSwimlane
         case topicSelector
+        case liveAudioSquaredSwimlane
```

Comment:
```suggestion
        case liveAudioSwimlane
```

Maybe better name, as it's only for audio livestreams, contrary to `liveMediaSwimlane` which is for both video and audio livestreams.